generated/nidaqmx/system/system.py

```diff
@@ -348,6 +349,7 @@ def get_digital_pull_up_pull_down_states(self, device_name):
 
         return power_up_states
 
+    @deprecation.deprecated(deprecated_in="0.7.0", details="Use set_analog_power_up_states_with_output_type instead.")
```

Comment:
0.7.0 is released, I think you mean 0.8.0?

Reply:
Was not sure of the next version number, hence left with 0.7.0. Will update to 0.8.0